core/src/test/scala/org/allenai/common/EnumSpec.scala

```diff
@@ -4,15 +4,11 @@ import org.allenai.common.testkit.UnitSpec
 
 import spray.json._
 
-class EnumSpec extends UnitSpec {
+import java.io.{ FileOutputStream, FileInputStream }
```

Comment:
Autoimporting is not possible in scalariform due to the fact that the order of import statements matters - you can import from local symbols - and scalariform is not a full compiler. However, it would be easy to add a script which did this heuristically, and would be guaranteed correct without relative imports.

@markschaake - I was referring to the two `import java.io.{ }` blocks. They should be merged, yes?